tests/org.eclipse.passage.seal.demo.tests/resource/anti-human-magic.product_0.2.1.keys_xmi

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="ASCII"?>
```

Comment:
Look at how licenses are stored. There should probably be a `resource/keys` folder for keys